src/test/datascience/interactive-common/randomBytes.functional.test.ts

```diff
@@ -0,0 +1,29 @@
+import { ISystemPseudoRandomNumberGenerator } from '../../../client/datascience/types';
+import { assert } from 'chai';
+import { DataScienceIocContainer } from '../dataScienceIocContainer';
+
+suite('DataScience - RandomBytes', () => {
```

Comment:
This seems like it could be a unit test? It doesn't need to run as part of other functional tests.

Reply:
Instead of using the container, you'd just instantiate the Pseudo random number generator.

Reply:
I wanted to test with a real process, not a mocked one, and I was under the impression that we mock dependencies (especially resource-intensive dependencies) in unit tests. If it's OK to use real processes I can do that!

Reply:
Blech just noticed the functional test is running with a mocked process anyway. Will redo the test